code/_onclick/adjacent.dm

```diff
@@ -94,6 +94,11 @@ Quick adjacency (to turf):
 /obj/item/Adjacent(atom/neighbor, recurse = 1)
 	if(neighbor == loc || (loc && neighbor == loc.loc))
 		return TRUE
+
+	// Internal storages have special relationships with the object they are connected to and we still want two depth adjacency for storages
+	if(istype(loc.loc, /obj/item/storage/internal) && recurse > 0)
```

Comment:
loc?.loc or similar
this would runtime if loc is null, which is likely a thing since there's a check for it above

Reply:
TRUE, will update 🙏